src/Faker/Provider/Base.php

```diff
@@ -263,7 +263,7 @@ public static function shuffle($arg = '')
      * @param array $array The set to shuffle
      * @return array The shuffled set
      */
-    public static function shuffleArray($array = array())
+    public static function shuffleArray(array $array = array())
```

Comment:
@GrahamCampbell , but this method willcall en php error in case non array passed, + param type is written in phpdoc ?